protocol/src/main/java/net/md_5/bungee/protocol/DefinedPacket.java

```diff
@@ -56,6 +56,22 @@ public static String readString(ByteBuf buf, int maxLen)
         return s;
     }
 
+    public static void skipString(ByteBuf buf)
+    {
+        skipString( buf, Short.MAX_VALUE );
+    }
+
+    public static void skipString(ByteBuf buf, int maxLen)
```

Comment:
Are to methods for this necessary?
As we skip the bytes anyway i think we don't need a extra limit and just skip the len